### ISSUE TITLE
model convert tensorRT

### ISSUE THREAD

**Issue by Winnie202** [dvlab-research/Entity#39]
Thanks for your work。Can the  Cropformer-Swin model be converted to tensorRT? How can I improve the speed of predict? I am currently testing 50 images using NVIDIA GeForce RTX 3090, which takes 15 minutes.

**Comment by qqlu** [comment]
Hi

We will try to convert the CropFormer to the TensorRT. However, I am not sure costing 15 minutes for 50 images is the normal speed.  Is your image very large? 

Lu

**Comment by Winnie202** [comment]
> Hi
> 
> We will try to convert the CropFormer to the TensorRT. However, I am not sure costing 15 minutes for 50 images is the normal speed. Is your image very large?
> 
> Lu

yes, the resolution 2048*3072. In addition, may I ask when the conversion to tensorrt can be realized
